src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -901,7 +905,6 @@ export default class FilingHistoryList extends Mixins(
       height: unset !important;
       min-width: unset !important;
       padding: 0.25rem !important;
-      color: $app-blue
```

Comment:
I moved this to the v-btn with class `menu-btn`. It seems to make the elements more consistent in the template.